packages/extension/src/background.ts

```diff
@@ -34,7 +45,7 @@ chrome.runtime.onConnect.addListener((port): void => {
   }, 250e3);
 
   // message and disconnect handlers
-  port.onMessage.addListener((data: TransportRequestMessage<keyof RequestSignatures>) => handlers(data, port));
+  port.onMessage.addListener((data: TransportRequestMessage<keyof RequestSignatures>) => handlers(data, port, ports));
```

Comment:
Suggestion from pair review:
```suggestion
  // Long comment with vibrant emoticons saying not to use the port directly so that the port reference is always fresh
  const getPort(port: 'content' | 'extension') => port === 'content' ? ports.contentPort : ports.extensionPort

  port.onMessage.addListener((data: TransportRequestMessage<keyof RequestSignatures>) => handlers(data, port, getPort));
```